modules/data.atmosphere/R/metgapfill.R

```diff
@@ -335,6 +335,7 @@ metgapfill <- function(in.path, in.prefix, outfolder, start_date, end_date, lst=
     nc_close(nc)
     
     if (length(error) > 0) {
+      file.remove(new.file)
```

Comment:
If you remove the file, the resuls will not be there. Maybe rename the file to paste(new.file, "bad", sep=".")